src/App.js

```diff
@@ -1,27 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
+import './Sass/app.css';
 
 function App() {
```

Comment:
- [ ] Please make sure you are having class based component in App.js as requested in project requested.